src/actions.test.js

```diff
@@ -120,248 +120,31 @@ describe('action creators', () => {
   });
 });
 
-describe('thunk action creator requestStatus', () => {
-  const releaseInfo = {
-    channel: 'release',
-    product: 'firefox',
-    version: '51.0',
-    checks: [
-      {
-        title: 'some test',
-        url: 'some url',
-      },
-    ],
-  };
-  it('requests the version given', async () => {
-    const mockCheckResult = {
-      status: 'exists',
-      message: 'check succesful',
-      link: 'some link',
-    };
-    // Mock the pollbot API calls.
-    const module = require('./PollbotAPI');
-    module.getReleaseInfo = jest.fn(() => releaseInfo);
-    module.checkStatus = jest.fn(() => mockCheckResult);
-
-    // Version 51.0 requested, version 50.0 in the state.
-    let expectedActions = [
-      {type: 'SET_VERSION', version: '51.0'},
-      {
-        releaseInfo: releaseInfo,
-        type: 'UPDATE_RELEASE_INFO',
-      },
-      {
-        result: mockCheckResult,
-        title: 'some test',
-        type: 'ADD_CHECK_RESULT',
-      },
-    ];
-
-    const store = mockStore({
-      version: '50.0',
-      checkResults: {},
+describe('sagas action creator', () => {
+  it('handles a REQUEST_ONGOING_VERSIONS action for requestOngoingVersions', () => {
+    expect(requestOngoingVersions()).toEqual({
+      type: REQUEST_ONGOING_VERSIONS,
     });
```

Comment:
Is it really valuable to test these functions?

Reply:
No idea, it's always the question with unit tests...